Blish HUD/GameServices/OverlayService.cs

```diff
@@ -229,18 +223,10 @@ public class OverlayService : GameService {
             HandleEnqueuedUpdates(gameTime);
 
             if (GameService.GameIntegration.IsInGame) {
-                _lastTacoCheckTime += gameTime.ElapsedGameTime.TotalSeconds;
-
-                if (_lastTacoCheckTime > 3) {
-                    Process[] tacoApp = Process.GetProcessesByName("GW2TacO");
-
-                    if (tacoApp.Length > 0) {
-                        CornerIcon.LeftOffset = 36 * (_clientType == Gw2ClientContext.ClientType.Chinese ? 2 : 1);
-                    } else {
-                        CornerIcon.LeftOffset = _clientType == Gw2ClientContext.ClientType.Chinese ? 36 : 0;
-                    }
-
-                    _lastTacoCheckTime = 0;
+                if (GameIntegration.TacO.TacOIsRunning) {
+                    CornerIcon.LeftOffset = 36 * (_clientType == Gw2ClientContext.ClientType.Chinese ? 2 : 1);
+                } else {
+                    CornerIcon.LeftOffset = _clientType == Gw2ClientContext.ClientType.Chinese ? 36 : 0;
                 }
```

Comment:
I need some help here, why is it 36 and not `CornerIcon.ICON_SIZE`?
In addition, following https://discordapp.com/channels/384735285197537290/603928533584183310/736854069028585472 I would like to simplify this code.

What do you think about this:
```cs
var offset = _clientType == Gw2ClientContext.ClientType.Chinese ? 1 : 0;
offset += GameIntegration.TacO.TacOIsRunning ? 1 : 0;

CornerIcon.LeftOffset = offset * 36;
```

Reply:
We found early on that the TacO icon did not look good with our standard static offset.  It needed a manual adjustment. 

I'll look at simplifying that section.

Reply:
Done!